src/api/plex.ts

```diff
@@ -112,8 +113,20 @@ export const allMovies = (async () => {
     }
 
     const libraryData: PlexMediaContainer<PlexVideo> = await req.json()
+    let metadata = libraryData.MediaContainer.Metadata
+
+    if (COLLECTION_FILTER !== '') {
+      const collectionFilter = COLLECTION_FILTER.split(',')
+      metadata = metadata.filter(x => {
```

Comment:
Can you rename `x` either to `_`, or something longer and more descriptive?

Other than that this MR LGTM.

Reply:
I renamed it, LMK if it is ok